eval.py

```diff
@@ -113,6 +121,9 @@ def parse_args(argv=None):
                         help='When displaying / saving video, draw the FPS on the frame')
     parser.add_argument('--emulate_playback', default=False, dest='emulate_playback', action='store_true',
                         help='When saving a video, emulate the framerate that you\'d get running in real-time mode.')
+    parser.add_argument('--num_cores', default=None, help="The num of cores to use (supported only with deepsparse), defaults to None")
+    parser.add_argument('--warm_up_iterations', default=0, type=int, help="The num of warm up iterations to run the engine for, defaults to None")
```

Comment:
let's support num_iterations as well. can default to current behavior, but otherwise, should loop the dataset until we hit the desired number of iterations

Reply:
Noted!

Reply:
Added